### ISSUE TITLE
HTMLRenderer: newlines inside paragraphs aren't counted as whitespace

### ISSUE THREAD

**Issue by Minoru** [akrennmair/newsbeuter#312]
The following HTML:
```
<p>one <a href="http://example.com/">two</a>
three</p>
```
is rendered as:
```
one two[1]three
```

Firefox adds a space between "two" and "three", which is indeed the behaviour I would expect.

Need to look this up in HTML specs.

**Comment by tsipinakis** [comment]
This is fixed with dcc6403a28c9c8fa586826b059083da2a33313be

**Comment by Minoru** [comment]
You're right on both points. Hmm, wonder what made me file this bug… Anyway, I'm happy to close it :)
